ctapipe/io/hdf5eventsource.py

```diff
@@ -243,7 +250,15 @@ def obs_ids(self):
         return list(np.unique(self.file_.root.dl1.event.subarray.trigger.col("obs_id")))
 
     @property
-    def simulation_config(self):
```

Comment:
We should probably add `simulation_config(s)` to the `EventSource` as interface, returning `None` for observations?

Reply:
Yes, to be symmetric this is a good idea.